packages-exp/installations-exp/src/api/delete-installations.ts

```diff
@@ -24,6 +24,7 @@ import { FirebaseInstallations } from '../interfaces/public-types';
 
 /**
  * Deletes the Firebase Installation and all associated data.
+ * @param installations - The Installations instance
```

Comment:
Is "Installations" a literal Fei?

If so, we should backtick it, here and throughout (wherever it is a literal).

Reply:
It's the product name, like Firebase App